sdk/confidentialledger/azure-security-confidentialledger/src/samples/java/com/azure/security/confidentialledger/CreateOrUpdateUser.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.confidentialledger;
+
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+
+public class CreateOrUpdateUser {
+    public static void main(String[] args) {
+        ConfidentialLedgerClient confidentialLedgerClient =
+                new ConfidentialLedgerClientBuilder()
+                        .credential(new DefaultAzureCredentialBuilder().build())
+                        .ledgerUri("https://my-ledger.confidential-ledger.azure.com")
+                        .buildClient();
+        // BEGIN:com.azure.security.confidentialledger.generated.createorupdateuser.createorupdateuser
```

Comment:
I feel like I might be missing something... I don't see any module that has "generated" in it or where the snippet itself is used.

Do we need to comment these (all the samples) with BEGIN/END if they're not used elsewhere?

Reply:
We do not; I was just trying to follow the protocol. I will remove it now!